### ISSUE TITLE
[BUG]: @toss/date cannot be imported properly

### ISSUE THREAD

**Issue by C17AN** [toss/slash#62]
### **Package Scope** 

Package name: `@toss/date`

### **Describe the bug**

When trying to use utils from `@toss/date` package, it throws error below.

```
Module not found: Error: Can't resolve 'date-fns/locale/ko' in '/Users/chanmin/react-test/node_modules/@toss/date/esm'
Did you mean 'index.js'?
BREAKING CHANGE: The request 'date-fns/locale/ko' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/26535030/196001113-c95ec7c7-9d2e-4f32-a5b3-c9482621308b.png">

### **Expected behavior**
`@toss/date` should be available without any tweak, but this package cannot be imported as expected.

### **To Reproduce** 

1. install `@toss/date`
2. Use any utils from `@toss/date`
3. Run / Build fails.

### **Possible Solution** 

Add `.js` extension on date-fns locale import

**AS-IS**

```
// packages/common/date/src/date.ts

// eslint-disable-next-line import/no-duplicates
import { format as _format, isAfter, isBefore, isEqual, isValid } from 'date-fns';
import locale from 'date-fns/locale/ko';  // => This import statement should be changed (to explicitly provide extension)
```

**TO-BE**

```
// eslint-disable-next-line import/no-duplicates
import { format as _format, isAfter, isBefore, isEqual, isValid } from 'date-fns';
import locale from 'date-fns/locale/ko/index.js';
```

### **Additional context** 

**Quotation**
- [https://bobbyhadz.com/blog/node-js-error-err-unsupported-dir-import](https://bobbyhadz.com/blog/node-js-error-err-unsupported-dir-import)
- [https://stackoverflow.com/questions/64449464/error-err-unsupported-dir-import-directory-import-when-attempting-to-start-no](https://stackoverflow.com/questions/64449464/error-err-unsupported-dir-import-directory-import-when-attempting-to-start-no)
- [https://toss.tech/article/commonjs-esm-exports-field](https://toss.tech/article/commonjs-esm-exports-field)

**Comment by raon0211** [comment]
Was fixed with #63, and was merged to upstream. We will release the new version soon!

**Comment by C17AN** [comment]
> Was fixed with #63, and was merged to upstream. We will release the new version soon!

Thanks @raon0211 , have great day! 🙂
